examples/js/fetch-ohlcv.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const ccxt = require ('../../ccxt.js');
+function initExchange (exchangeName) {
+    return new ccxt[exchangeName]();
+}
+
+// AUTO-TRANSPILE //
+
+async function myfunc () {
+    const exchange = initExchange ('binance');
```

Comment:
not sure if this is flexible enough to handle things like custom options, credentials etc 🤔

Reply:
absolutely that's not flexible enough. that's why i said in top post "this is just start" and any further notes & edits are needed. i'll update it now but plz add your suggestions too. again, we should start from something to make it grow.

Reply:
@ttodua sure this is already a great start 🙂 

second question, should we consider transpile this to python/php async?

Reply:
fixed the transpiler, so there is no need to be separate `initExchange` (the class instantiation is now being transpiled natively)

Reply:
about async - i need your thoughts, how we should do it. there are basic examples, that only calls a single endpoint (and in those cases, there is no need of async imho), however, for the sake of the completeness, should we transpile every js file, i.e.:
`filename.js`
--> `filename.php` & `filename-async.php`
--> `filename.py` & `filename-async.py`
?

Reply:
also, there are some gotchas, i.e. in js you can call `async` method without `await`:
```
async function x() {

}
x();
```
however, in py it needs to have `await x()` otherwise throws error. So, we are not fully free to write JS examples in such manner. or do you think of workaround?

Reply:
added also async-php, however, both py/php are structurally different and this will need more edits to become better. However, at this moment, as you can test it out too, it does work with `fetch-ohlcv.js` example.

Reply:
> --> filename.php & filename-async.php
@ttodua, actually I think we should place all transpiled examples in a different folder, what do you think? By doing that it's easy to tell if some example is automatically transpiled or not

> example() to be name of the sample function
humm why is that? 

> exchange variable to be the main holder of instantiated
yeah this is because of the .close right? yeah for this  I don't see a workaround

Reply:
@carlosmiei I don't know which is best - to mix the transpilable snippets in main JS folder with others, or not. 
imo, it can be in same folder, as is, and the `AUTO-TRANSPILE` head-comment makes it clear for users, that the file is transpiled one.
@kroitor your thoughts are needed.

Reply:
2)

Reply:
it also needs some brainstorming, to think how to design the rules for writing the transpilable example. to refer to `await ex.close()`  problem in py, or async method recognition and their correct transpilation into py/php (as their async syntax is different from JS async/await flow...